spirv_cross.cpp

```diff
@@ -2378,8 +2378,10 @@ SPIREntryPoint &Compiler::get_entry_point()
 bool Compiler::interface_variable_exists_in_entry_point(uint32_t id) const
 {
 	auto &var = get<SPIRVariable>(id);
-	if (var.storage != StorageClassInput && var.storage != StorageClassOutput)
-		SPIRV_CROSS_THROW("Only Input and Output variables are part of a shader linking interface.");
+	if (var.storage != StorageClassInput && var.storage != StorageClassOutput &&
+	    var.storage != StorageClassUniformConstant)
```

Comment:
The SPIR-V spec identifies only `StorageClassInput` and `StorageClassOutput` variables in `OpEntryPoint`. Can you clarify how a struct with`StorageClassUniformConstant` arrives in `OpEntryPoint`? Do you have example shader snippets you can share?